src/types/preact-render-to-string.d.ts

```diff
@@ -0,0 +1,24 @@
+// This local type definition is used to address an incorrect one from
+// preact-render-to-string.
```

Comment:
Would you mind filing an issue upstream with the error and referencing it here. I think it should be easy enough to address.

Reply:
On my way